central/compliance/handlers/csv.go

```diff
@@ -199,7 +199,7 @@ func CSVHandler() http.HandlerFunc {
 				output.addAll(mcRow, controls, mcValue.GetControlResults())
 			}
 		}
-		output.Write(w, "compliance_export")
+		output.Write(w, "compliance_export.csv")
```

Comment:
`.csv` is added by the writer, if I'm not mistaken:
https://github.com/stackrox/stackrox/blob/4a9e5aa6621eb4bb06af6da737a9e85b4b6f9b68/pkg/csv/writer.go#L64